### PR TITLE
Tidy up range docs

### PR DESCRIPTION
**Pull request by bradcray**
While reviewing my the range docs after my changes to its behavior
last night, I found some pre-existing bugs that needed fixing,
that the operators were now being generated by chpldoc, and some
other opportunities for cleanup.  Summarizing, they were:

* the expected output from the alignedLow/High docs wasn't printing
  (and hadn't been in previous releases either)

* the signatures on the param/type queries in the spec used illegal
  syntax (though now they fall prey to the poor spacing when
  rendered by Sphinx on Chrome.

* I squashed the chpldocumentation of the operators because they're
  already described in the text of the spec, which does a better job
  of it.  That said, going forward, I think we'd like to find a way to
  integrate operator signatures into these sections of the spec so
  that people can see what the overloads are.  Not today's task though.

* I tried to clarify some wordings and descriptions.

* removed the "more descriptive" clause that I'd applied to the
  alignedLow/High queries yesterday because it sounded like more of
  a value judgement or preference than I'd intended.

* I more uniformly set ``true`` and ``false`` off using code
  formatting.

* I more uniformly started entries with "Returns" rather than "Return"

* I more uniformly ended descriptions with periods

### CODE REVIEW

File: modules/internal/ChapelRange.chpl [hunk 1]
```diff
@@ -757,10 +763,10 @@ module ChapelRange {
...
   }
 
-  /* Return true if the range has a first index, false otherwise.
-     Note that in the event that the range is stridable and at least
-     partially bounded, the return value will not (cannot) be a
-     `param`.
+  /* Returns ``true`` if the range has a first index, ``false``
+     otherwise.  Note that in the event that the range is stridable
+     and at least partially bounded, the return value will not
+     (cannot) be a `param`.
```
Comment (bmcdonald3) [inline modules/internal/ChapelRange.chpl:769]: I find this "(cannot)" to be slightly distracting from the message. It seems a bit unnecessary and would more likely cause someone to wonder "why not?" and detract from the message, rather than provide any actual value to a user. Either they understand why it can't be so or they don't, no reason to call it out in my opinion.

File: modules/internal/ChapelRange.chpl [hunk 2]
```diff
@@ -1075,8 +1084,8 @@ operator :(r: range(?), type t: range(?)) {
   }
 
   /*
-     If ``ind`` is a member of the range's represented sequence, returns
-     an integer giving the ordinal index of ind within the sequence
+     If ``ind`` is a member of the range's represented sequence, this returns
```
Comment (bmcdonald3) [inline modules/internal/ChapelRange.chpl:1087]: The phrasing of "this returns" seems unique here. I would choose to reword this to something along the lines of: "Returns an integer representing the ordinal index of ``ind`` within the sequence using zero-based indexing if ``ind`` is a member of the range's represented sequence. Otherwise, returns -1."

This makes it more consistent with the other docs and puts the `if`/`otherwise` closer together to make it easier to keep track of, but this is minor and I don't think needs to change, regardless of whatever the length of this message implies.

**Review by bmcdonald3 — APPROVED** [review]
Looks good, a few minor comments that don't need addressing.

File: modules/internal/ChapelRange.chpl (hunk 2 above)
Comment (bradcray) [reply modules/internal/ChapelRange.chpl]: Done, thanks!
